Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -901,7 +901,6 @@ class BrowserViewController: UIViewController {
             }
         })
         view.setNeedsUpdateConstraints()
-        urlBar.locationView.reloadButton.reloadButtonState = .disabled
```

Comment:
this line here set the reloadButtonStatus to **disabled**  and it's called each time when focusing out of the URL bar